packages/design/src/tokens/color.tokens.json

```diff
@@ -0,0 +1,268 @@
+      },
+      "lightest": {
+        "$value": "{color.base.lime-.200}"
+      },
+      "dark": {
+        "$value": "{color.base.lime-.700}"
+      }
+    },
+    "yellowGreen": {
+      "$value": "{color.base.lime-.500}"
+    },
+    "yellowGreen-": {
+      "light": {
+        "$value": "{color.base.lime-.400}"
+      },
+      "lighter": {
+        "$value": "{color.base.lime-.300}"
+      },
+      "lightest": {
+        "$value": "{color.base.lime-.200}"
+      },
+      "dark": {
+        "$value": "{color.base.lime-.600}"
+      }
+    },
+    "yellow": {
+      "$value": "{color.base.yellow-.400}"
+    },
+    "yellow-": {
+      "light": {
+        "$value": "{color.base.yellow-.300}"
+      },
+      "lighter": {
+        "$value": "{color.base.yellow-.200}"
+      },
+      "lightest": {
+        "$value": "{color.base.yellow-.100}"
+      },
+      "dark": {
+        "$value": "{color.base.yellow-.600}"
+      }
+    },
+    "red": {
+      "$value": "{color.base.red-.600}"
+    },
+    "red-": {
+      "light": {
+        "$value": "{color.base.red-.400}"
+      },
+      "lighter": {
+        "$value": "{color.base.red-.300}"
+      },
+      "lightest": {
+        "$value": "{color.base.red-.200}"
+      },
+      "dark": {
+        "$value": "{color.base.red-.800}"
+      }
+    },
+    "grey": {
+      "$value": "{color.base.grey-.400}"
+    },
+    "grey-": {
+      "light": {
+        "$value": "{color.base.grey-.300}"
+      },
+      "lighter": {
+        "$value": "{color.base.grey-.200}"
+      },
+      "lightest": {
+        "$value": "{color.base.grey-.100}"
+      },
+      "dark": {
+        "$value": "{color.base.grey-.600}"
+      }
+    },
+    "overlay": {
+      "$value": "rgba({color.black-.rgb}, 0.32)"
+    },
+    "overlay-": {
+      "dimmed": {
+        "$value": "rgba({color.white-.rgb}, 0.6)"
+      }
+    },
+    "white": {
+      "$value": "rgba({color.white-.rgb}, 1)"
+    },
+    "black": {
+      "$value": "rgba({color.black-.rgb}, 1)"
+    },
+    "greyBlue": {
+      "$value": "{color.base.blue-.600}"
+    },
+    "greyBlue-": {
+      "light": {
+        "$value": "{color.base.blue-.500}"
+      },
+      "lighter": {
+        "$value": "{color.base.blue-.400}"
+      },
+      "lightest": {
+        "$value": "{color.base.blue-.200}"
+      },
+      "dark": {
+        "$value": "{color.base.blue-.800}"
+      }
+    },
+    "lightBlue": {
+      "$value": "{color.base.lightBlue-.500}"
+    },
+    "lightBlue-": {
+      "light": {
+        "$value": "{color.base.lightBlue-.400}"
+      },
+      "lighter": {
+        "$value": "{color.base.lightBlue-.300}"
+      },
+      "lightest": {
+        "$value": "{color.base.lightBlue-.200}"
+      },
+      "dark": {
+        "$value": "{color.base.lightBlue-.600}"
+      }
+    },
+    "purple": {
+      "$value": "{color.base.pink-.800}"
+    },
+    "purple-": {
+      "dark": {
+        "$value": "{color.base.pink-.900}"
+      }
+    },
```

Comment:
Yeah I broke the colors up into a few different files, you have the right idea.

baseColor.tokens - Any color token with a direct value. No aliases, this is the base file for everything.
color.tokens - Any token alias that is just a color name. Not semantic, not a base color, not feature specific.
semanticColor.tokens - Semantic colors. Convey meaning instead of color.
workflow.tokens -> Workflow/Feature specific colors, similiar to semanticColors but specific to Jobber Features.

I'm certainly open to other organizations, but that's how I structured it in my head.